docs/sources/administration/configuration.md

```diff
@@ -772,6 +772,22 @@ Refer to [Auth proxy authentication]({{< relref "../auth/auth-proxy.md" >}}) for
 
 Refer to [LDAP authentication]({{< relref "../auth/ldap.md" >}}) for detailed instructions.
 
+## [aws]
+
+You can configure core and external AWS plugins.
+
+### allowed_auth_providers
+
+Specify what authentication providers that the AWS plugins allow. For a list of allowed providers, refer to the data-source configuration page for a given plugin. If you configure a plugin by provisioning, only providers that are specified in `allowed_auth_providers` are allowed.
+
+Options: `default` (AWS SDK Default), `keys` (Access && key secret key), `credentials` (Credentials field), `ec2_IAM_role` (EC2 IAM Role)
+
+### assume_role_enabled
+
+Set to false to disable AWS authentication using an assumed role with temporary security credentials. Please refer to [AWS official docs](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html) for details about assume roles.
+
+If this option is disabled, the **Assume Role** and the **External Id** field will be removed from the AWS data source configuration page. If the plugin is configured using provisioning, it will be possible using an assumed role as long as `assume_role_enabled` is set to true.
```

Comment:
```suggestion
If this option is disabled, the **Assume Role** and the **External Id** field are removed from the AWS data source configuration page. If the plugin is configured using provisioning, it is possible to use an assumed role as long as `assume_role_enabled` is set to `true`.
```
Vet this for technical accuracy. I was not sure what "it" meant in, ", it will be possible using an assumed role..."

Reply:
Looks good, thanks!